ansible/roles/box/templates/config.ts

```diff
@@ -0,0 +1,116 @@
+import GossipSub from "libp2p-gossipsub";
```

Comment:
Does it need to be part of template?!

Reply:
It should not be - this is just a quick work around to disable NAT for now until we find a better solution.  I believe options are to enable/disable from our config or to find a way to trap the exception so that it does not bubble up and crash the app.

Reply:
So let it be. create a issue for it.
#194